logger/loggerApp.swift

```diff
@@ -13,7 +13,9 @@ struct loggerApp: App {
 
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            NavigationStack{
```

Comment:
This shouldn't be necessary. There is already a `NavigationStack` on `ContentView.swift:24`.